frameworks/cassandra/tests/conftest.py

```diff
@@ -0,0 +1,43 @@
+import json
+import os
+import random
+import string
+
+import pytest
+import shakedown
+
+## Commented out so it doesn't actually run.
+## Using the PR for comments and feedback, don't want the tests.
+#@pytest.fixture(scope='session', autouse=True)
+def configure_universe(request):
+    stub_urls = {}
+
+    # prepare needed universe repositories
+    stub_universe_urls = os.environ.get('STUB_UNIVERSE_URL')
+    if not stub_universe_urls:
+        return
+    for url in stub_universe_urls.split(' '):
+        package_name = 'testpkg-'
+        package_name += ''.join(random.choice(string.ascii_lowercase + string.digits) for _ in range(8))
```

Comment:
I'm going to change the variable name to `repo_name` because I think that's causing the confusion. There shouldn't need to be any coordination with the tests. This logic is identical to what's currently running via `run_tests.py`.